src/Quadrature.h

```diff
@@ -178,12 +182,20 @@ class LeonardPolarQuad: public Quadrature {
 class GLPolarQuad: public Quadrature {
 
 private:
+  /** the roots to the Legendre polynomial of degree _num_polar */
+  std::vector <double> _roots;
```

Comment:
I see what you're saying - but it seems like we can get the roots direction from the ``_thetas`` array. Also does this PR incorporate the weight correction?